Source/HeaderableTableDataDisplayManager.swift

```diff
@@ -39,8 +39,8 @@ open class BaseHeaderableTableDataDisplayManager: NSObject, HeaderableTableDataD
 
     // MARK: - Fileprivate properties
 
-    fileprivate var cellGenerators: [[TableCellGenerator]]
-    fileprivate var sectionHeaderGenerators: [TableHeaderGenerator]
+    var cellGenerators: [[TableCellGenerator]]
+    var sectionHeaderGenerators: [TableHeaderGenerator]
```

Comment:
I think there is no need `internal` access. Let's make it `public fileprivate(set)`